src/docfx/build/xref/IXrefSpec.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Docs.Build
+{
+    internal interface IXrefSpec
+    {
+        string Href { get; }
+
+        Document ReferencedFile { get; }
```

Comment:
nit: consider rename it to `DeclaringFile` to indicate that it is the file that declares this xref.